Translation/en/RedirectedResources/assets/abdata/h/list/71/personality_voice_c01/translation.txt

```diff
@@ -60,14 +60,14 @@
 お友達がおちんちんを舐めている所をこんな間近で見るなんて……=I can't believe I'm watching my friend lick your dick from so up close...
 私だって、あなたを気持ちよくさせます……=I can, pleasure you too...
 世間知らずでも、あなたのおちんちんの気持ちいいところなら知ってます……=I may be ignorant to the ways of the world but, all the sensitive spots on your dick are an entirely different story...
-//（ゆっくり舐められるのは、お好きですか？）=
-//（乳首……あなたの乳首っ……）=
-//（乳輪を……なぞるように……）=
-//（ずっと舐めていられるかもしれません……）=
-//（硬くなっていくのが、分かりますよ……）=
-//（ちゅぱちゅぱエッチな音が、とまりません）=
-//（私……舐めることが、好きなのかもしれません）=
-//（乳首で感じてるあなたの声、もっと聴きたいです）=
+（ゆっくり舐められるのは、お好きですか？）=(Do you like it when I, do it slowly?)
```

Comment:
I feel like there was a reason we skipped the subtitles in `()` but I can't remember.  Have we verified these lines are spoken?  Maybe that was a different game. If it turns out there's a reason to skip them we can comment them back out (and maybe add something to the `README`)

Reply:
There shouldn't be any reason to skip them, even if they are unused in the game they will just get ignored.

Reply:
I think it was a different game where the lines wrapped in `()` *were* used by the game, but were not transcriptions so we didn't want them showing up.